src/community/index.md

```diff
@@ -68,8 +68,7 @@ Our wonderful community has provided these resources.
 * [Dart Academy](https://dart.academy/) - Tutorials
   and articles written by the Dart community
 * [Dartosphere blog aggregator](http://www.dartosphere.org)
-* [Resources for Chinese developers](http://www.dartlang.cc/support/for-chinese.html)
-
+* [Chinese version of this site](http://dart.goodev.org/)
```

Comment:
Maybe this should be written in Chinese?

Reply:
That would be nice. I'll file a help wanted issue.